modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/introspection/describer/AnnotationsBasedDescriber.java

```diff
@@ -445,6 +448,20 @@ private void declareAnnotatedParameters(Class<?> annotatedType, ParameterizedDec
         return groups;
     }
 
+    /**
+     * Parameters inside a group annotated with {@link Exclusion} are all considered as optionals.
+     *
+     * @param clazz      of a {@link Field} considered a parameter container
+     * @param parameters contained inside the container
+     */
+    private void enforceOptionalIfExclusive(Class<?> clazz, Set<ParameterDeclarer> parameters)
+    {
+        if (getAnnotation(clazz, Exclusion.class) != null)
+        {
+            parameters.stream().forEach(p -> p.getDeclaration().setRequired(false));
```

Comment:
this is wrong. Exclusive behaviour should only be applied to the optional parameters of the annotated pojo. If a user says that something is required, then it's required. If the amount of optional parameters <=1 -> IllegalParameterModelDEfinitionException